ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/ccache/tasks/main.yml

```diff
@@ -42,6 +47,7 @@
     state: absent
   with_items:
     - /tmp/ccache.tar.gz
-    - /tmp/ccache-3.4.2
+    - /tmp/ccache-{{ ccacheVersion }}
+    - /home/{{ Jenkins_Username }}/ccache.tar
```

Comment:
Where is this being creasted? Not sure /home/jenkins/ccache.tar should ever exist

Reply:
I'm not sure either? I left it though to be safe

Reply:
I say delete it - especially now that the download, building and removal is all in one place it looks completely obsolete. I suspect at one point in the past we tried to extract from the tar.gz into a .tar and so it's historic as we don't do that any more. If you can do that I'll happily approve :-)

Reply:
removed